website/src/pages/v3/comparison.mdx

```diff
@@ -95,6 +95,31 @@ We believe GraphQL Yoga is the best Apollo Federation Server out there.
 
 The migration from Apollo Server to Yoga Server is easy and you can find [instructions here](https://www.the-guild.dev/graphql/yoga-server/v3/migration/migration-from-apollo-server).
 
+## GraphQL Yoga and express-graphql
+
+### Maintainability
+
+The express-graphql project has been unmaintained for many years now.
+
+Also, the project started as a small example of how to use GraphQL with a server and was never aimed to be used in production.
+
+It simply grew by itself without ever maturing in terms of code and ownership.
+
+### Better in everything else
+
+If you check out the [comparison above to Apollo Server](https://www.the-guild.dev/graphql/yoga-server/v3/comparison#graphql-yoga-and-apollo-server), everything also applies also to the comparison with `express-graphql`. Same as with the Apollo Server Comparison, there isn't a single thing that express-graphql and Apollo Server are doing better than Yoga, including the express integration:
+
+* [No Framework-specific Packages](https://www.the-guild.dev/graphql/yoga-server/v3/comparison#no-framework-specific-packages)
```

Comment:
We should make all these links relative instead of full domain name here

Reply:
Done